trace/trace_opentelemetry/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
```

Comment:
How is this sample intended to be used? That informs the type of sample we should write. Quickstarts are usually `package main` and have a `func main()`. That is the entrypoint to the program and what is run when you build+run the executable.

Reply:
The sample shows how to install and import OpenTelemetry.